packages/elements-core/package.json

```diff
@@ -0,0 +1,126 @@
+{
+  "name": "@stoplight/elements-core",
+  "version": "7.0.0-beta.1",
```

Comment:
Since business decision was to start numbering `dev-portal` from `1.0.0` anyways, perhaps we could start from `1.0.0` here as well? :thinking:

Reply:
What? :o Do we start dev portal from 1.0.0? @philsturgeon

Reply:
Yep yep! It’s the first version of this extension package. Elements and Elements Dev Portal are different things and don’t need to have matching numbers.

Reply:
I don’t mind what you number the core and utils as. They’re not user facing so could be v7.0

Reply:
Let's go with `elements-core@1.0.0` and let them both diverge over time.